docs/developer-docs/latest/development/typescript.md

```diff
@@ -67,126 +67,68 @@ To experience TypeScript-based autocomplete while developing Strapi applications
 3. Choose `runLifecyclesfunction` from the list.
 4. When the `strapi.runLifecyclesFunctions` method is added, a list of available lifecycle types (i.e. `register`, `bootstrap` and `destroy`) are returned by the code editor. Use keyboard arrows to choose one of the lifecycles and the code will autocomplete.
 
-## Develop a plugin using TypeScript
-
-New plugins can be generated following the [plugins development documentation](/developer-docs/latest/development/plugins-development.md). There are 2 important distinctions for TypeScript applications:
-
-- After creating the plugin, run `yarn install` or `npm run install` in the plugin directory `src/admin/plugins/[my-plugin-name]` to install the dependencies for the plugin.
-- Run `yarn build` or `npm run build` in the plugin directory `src/admin/plugins/[my-plugin-name]` to build the admin panel including the plugin.
-
-::: note
-It is not necessary to repeat the `yarn install` or `npm run install` command after the initial installation. The `yarn build` or `npm run build` command is necessary to implement any plugin development that affects the admin panel.
-:::
-
-## Start Strapi programmatically
-
-Instantiating Strapi programmatically in a TypeScript project requires additional configurations to load everything correctly. The primary difference for TypeScript programmatic use is that the codebase and compiled code are stored in separate directories, whereas the same directory is used to read and write in native JavaScript.
-
-### Understand programatic use
-
-When instantiating Strapi programmatically using the default export of `@strapi/strapi`, different parameters such as the `app` directory and the `dist` directory can be passed. The `app` directory is the project root directory and `dist` is a subdirectory of `app`. The `dist` directory represents the compiled project, with the same directory structure as the `app` directory, and the `app` directory represents the codebase (TypeScript or JavaScript). The main differences are:
+## Generate typings for project schemas
 
-- When using the Content-type Builder to create, update, delete content types (or any other service that creates files), Strapi uses the `app` folder to write the files.
-- When reading what content types exist on the system Strapi reads the `dist` folder.
-- When the `develop` command is used only the `app` directory is watched. In TypeScript, a compilation will be triggered when a change is detected in the `app` directory and the output will be written to the `dist` directory.
+To generate typings for your project schemas use the Strapi CLI and the command `content-types:generate-types`. The `content-types:generate-types` command creates the file `schemas.d.ts`, at the project root, which stores the schema typings. The optional `--verbose` flag returns the project type definitions in the terminal.
```

Comment:
`ts:generate-types` :)

Reply:
Also, the list of flags and what they do is explained [here](https://github.com/strapi/strapi/pull/13460)

![image](https://user-images.githubusercontent.com/25851739/180231470-504761d6-a085-422e-a5ba-e1321a2d8338.png)

`--silent` completely removes all the logs in the terminal
`--verbose` display a detailed table of the generated schemas (but not the exact definitions)